src/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchCryptoData } from './redux/cryptoSlice';
+import CryptoList from './components/CryptoList';
+import CryptoDetails from './components/CryptoDetails';
+import Navbar from './components/Navbar';
+
+function App() {
+  const dispatch = useDispatch();
+
+  // Fetch data when the component mounts
+  useEffect(() => {
```

Comment:
- [optional] please remember, we are replicating someone's design and personalizing the content. Kindly add the arrows on each card, so it follows the original design. 
yours
![Screenshot_32](https://github.com/CVILLA09/my-crypto-metrics/assets/65298048/fe391a72-ba4a-4b3a-8681-f0c12373f0c4)
original
![Screenshot_31](https://github.com/CVILLA09/my-crypto-metrics/assets/65298048/99ad0a7b-94a7-4df9-a1a3-56f4d38f4917)